x-pack/plugins/elastic_assistant/server/routes/knowledge_base/get_knowledge_base_status.ts

```diff
@@ -61,12 +61,11 @@ export const getKnowledgeBaseStatusRoute = (router: ElasticAssistantPluginRouter
 
           const indexExists = true; // Installed at startup, always true
           const pipelineExists = true; // Installed at startup, always true
-          const modelExists = await kbDataClient.isModelInstalled();
           const setupAvailable = await kbDataClient.isSetupAvailable();
           const isModelDeployed = await kbDataClient.isModelDeployed();
 
           const body: ReadKnowledgeBaseResponse = {
-            elser_exists: modelExists,
+            elser_exists: isModelDeployed,
```

Comment:
We use `elser_exists` to see the KB readiness and `is model deployed` represents that more accurate than `is model installed` state.